TestProject/Controllers/AccountController.cs

```diff
@@ -43,7 +43,30 @@ public async Task<IActionResult> GoogleResponse()
             };
             user = await _userService.AddUserAsync(user);
 
-            return Json(user);
+            return Redirect("/");
+        }
+
+        [Route("logout")]
+        public async Task<IActionResult> Logout()
+        {
+            await HttpContext.SignOutAsync(CookieAuthenticationDefaults.AuthenticationScheme);
+            return Redirect("/");
+        }
+
+        [Route("getUser")]
+        public async Task<ActionResult<UserLoginModel>> GetUser()
+        {
+            var result = await HttpContext.AuthenticateAsync(CookieAuthenticationDefaults.AuthenticationScheme);
+            var claims = result.Principal?.Identities.FirstOrDefault().Claims;
+            if (claims != null)
```

Comment:
this check is not needed in case user authenticated, that should be checked by auth filter.